<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VIPRegister</class>
 <widget class="QWidget" name="VIPRegister">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>450</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="name_LEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="register_Btn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>91</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ADMUI3Lg</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QPushButton" name="close_Btn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>250</y>
     <width>91</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ADMUI3Lg</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="no_LEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>100</y>
     <width>341</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ADMUI3Lg</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>会员名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ADMUI3Lg</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>卡号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ADMUI3Lg</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>日期</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="date_LEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
